src/EditorFeatures/CSharpTest/Braces/AddBracesTests.cs

```diff
@@ -600,7 +677,358 @@ static void Main()
 }",
 
             index: 0,
-            compareTokens: false);
+            compareTokens: false,
+            options: AddBraces());
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddBraces)]
+        public async Task FireForFixedWithoutBraces()
+        {
+            await TestAsync(
+            @"
+class Point 
+{ 
+    public int x;
+    public int y; 
+}
+
+class Program
+{
+    unsafe static void TestMethod()
+    {
+        var pt = new Point();
+        [|fixed|] (int* p = &pt.x)
+            *p = 1;
+    }
+}",
+
+   @"
+class Point 
+{ 
+    public int x;
+    public int y; 
+}
+
+class Program
+{
+    unsafe static void TestMethod()
+    {
+        var pt = new Point();
+        fixed (int* p = &pt.x)
+        {
+            *p = 1;
+        }
+    }
+}",
+
+            index: 0,
+            compareTokens: false,
```

Comment:
compareTokens is for formatting tests.  if these tests are not actually trying to verify formatting, we should remove this.